keps/sig-storage/20190709-csi-snapshot.md

```diff
@@ -89,28 +92,150 @@ The rest of the document includes required information missing from the original
 * Snapshot feature is used as a basic building block in other advanced applications. 
 * Feature deployed in production and have gone through at least one K8s upgrade.
 
-## Changes
-
-### Changes Implemented
+## Snapshot Beta
+
+### API Changes
+
+A number of changes were made to the Kubernetes volume snapshot API between alpha to beta. These changes are not backward compatible and the alpha API is no longer supported. The purpose of these changes was to make API definitions clear and easier to use.
+
+The following changes are made:
+
+* DeletionPolicy is now a required field rather than optional in both VolumeSnapshotClass and VolumeSnapshotContent. This way the user has to explicitly specify it, leaving no room for confusion.
+* VolumeSnapshotSpec has a new required Source field. Source may be either a PersistentVolumeClaimName (if dynamically provisioning a snapshot) or VolumeSnapshotContentName (if pre-provisioning a snapshot).
+* VolumeSnapshotContentSpec also has a new required Source field. This Source may be either a VolumeHandle (if dynamically provisioning a snapshot) or a SnapshotHandle (if pre-provisioning volume snapshots).
+* VolumeSnapshotStatus now contains a BoundVolumeSnapshotContentName to indicate the VolumeSnapshot object is bound to a VolumeSnapshotContent.
+* VolumeSnapshotContentnow contains a Status to indicate the current state of the content. It has a field SnapshotHandle to indicate that the VolumeSnapshotContent represents a snapshot on the storage system.
```

Comment:
"It has a field SnapshotHandle which is the unique identifier of a snapshot on the storage system"